src/main/java/core/basesyntax/FieldXY.java

```diff
@@ -1,19 +1,37 @@
 package core.basesyntax;
 
-/**
- * <p>На игровом поле находится робот. Позиция робота на поле описывается двумя
- * целочисленным координатами: X и Y. 
- * Ось X смотрит слева направо, ось Y — снизу вверх.(Помните, как рисовали 
- * графики функций в школе?)
- * В начальный момент робот находится в некоторой позиции на поле.
- * Также известно, куда робот смотрит: вверх, вниз, направо или налево.
- * Ваша задача — привести робота в заданную точку игрового поля.
- * Реализуйте метод moveRobot()</p>
- */
 public class FieldXY {
 
     public void moveRobot(Robot robot, int toX, int toY) {
-        //your code
+        if (robot.getX() < toX) {
+            while (robot.getDirection() != Direction.RIGHT) {
+                robot.turnRight();
+            }
+            while (robot.getX() != toX) {
+                robot.stepForward();
+            }
+        } else if (robot.getX() > toX) {
```

Comment:
```suggestion
        } else {
```
don't create redundant checks